diff.py

```diff
@@ -416,6 +418,14 @@ class Compress:
     same_instr: bool
 
 
+class DiffType(enum.Enum):
```

Comment:
Mode could be clearer than Type. It's still imperfect, because there's multiple modes being configured -- also e.g. -o vs --elf vs neither -- but whatever, it's not user-exposed and the usage is clear from context.